RedditSharp/Multi/Multi.cs

```diff
@@ -0,0 +1,362 @@
+        /// <returns>A MultiData containing the description for the found Multi</returns>
+        public MultiData GetMultiDescription(string path)
+        {
+            var request = WebAgent.CreateGet(string.Format(GetMultiDescriptionPathUrl, path));
+            var response = request.GetResponse();
+            var json = JToken.Parse(WebAgent.GetResponseString(response.GetResponseStream()));
+            var result = new MultiData(Reddit, json, WebAgent, false);
+            return result;
+        }
+
+        /// <summary>
+        /// Retrieve the information for a given subreddit in a Multi
+        /// </summary>
+        /// <param name="path">URL path to use</param>
+        /// <param name="subreddit">Subreddit name to get information for</param>
+        /// <returns>A MultiSubs element containing the information for the searched subreddit</returns>
+        public MultiSubs GetSubInformation(string path, string subreddit)
+        {
+            var request = WebAgent.CreateGet(string.Format(GetMultiSubUrl, path,subreddit));
+            var response = request.GetResponse();
+            var json = JToken.Parse(WebAgent.GetResponseString(response.GetResponseStream()));
+            var result = new MultiSubs(Reddit, json, WebAgent);
+            return result;
+        }
+
+        /// <summary>
+        /// Rename a Multi based on the given information
+        /// </summary>
+        /// <param name="displayName">New Display name for the Multi</param>
+        /// <param name="pathFrom">Original URL path of the Multi</param>
+        /// <param name="pathTo">New URL path of the Multi</param>
+        /// <returns>A String containing the new Multi information</returns>
+        public string RenameMulti(string displayName, string pathFrom, string pathTo)
+        {
+            if (Reddit.User == null)
+                throw new AuthenticationException("No user logged in.");
+            var request = WebAgent.CreatePost(PostMultiRenameUrl);
+            var stream = request.GetRequestStream();
+            WebAgent.WritePostBody(stream, new
+            {
+                display_name = displayName,
+                from = pathFrom,
+                to = pathTo,
+                uh = Reddit.User.Modhash
+            });
+            stream.Close();
+            var response = request.GetResponse();
+            var data = WebAgent.GetResponseString(response.GetResponseStream());
+            return data;
+        }
+
+        /// <summary>
+        /// Adds a Subreddit to the given Multi
+        /// </summary>
+        /// <param name="path">URL Path of the Multi to update</param>
+        /// <param name="subName">Name of the subreddit to add</param>
+        /// <returns>A String containing the information of the updated Multi</returns>
+        public string PutSubMulti(string path, string subName)
+        {
+            if (Reddit.User == null)
+                throw new AuthenticationException("No user logged in.");
```

Comment:
Enclose this in curly braces to be more specific about the `if` in case of future changes. I'm pretty sure this style is used elsewhere in the repo, but I'd like to move away from it if possible.